src/components/ha-chip-set.ts

```diff
@@ -21,7 +21,7 @@ declare global {
 
 @customElement("ha-chip-set")
 export class HaChipSet extends LitElement {
-  @property() public items = [];
+  @property() public items: string[] = [];
```

Comment:
I know this is existing code, but isn't it weird that we allow passing in strings here instead of slotted components? Now it's not possible to style each individual chip and we need to have our own `chip-clicked` event (like we had `actionText` / `action-clicked` in `ha-alert`).

Reply:
The CSS that mdc chip set brings in seems to be 

```css
padding: 4px;
display: flex;
flex-wrap: wrap;
box-sizing: border-box;
```

Reply:
Oh, definitely. In <https://github.com/home-assistant/frontend/pull/10410> I added slots to the  `chip-set` as well, so can do that here